sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,11 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self._eval_is_real():
```

Comment:
Alright! Will do. Can you please re-run the Travis build? All tests are working fine on my computer.